### PR TITLE
Kumar code review

### PR DESCRIPTION
**Pull request by kummy98**
i due file di cui fare la code review sono la main activity e l'empty activity

### CODE REVIEW

**Review by MarcoTomma — CHANGES_REQUESTED** [review]
- **Miscellaneous**
Some access modifiers are superfluous like "private" ;
You should also keep attention to constants using the caps lock (line: 36, 37, 38) ;
The keyword "<#>" is hardwired (line: 163) ;
- **Methods**
getCoordinates (line: 118) does too much things (3): it instantiates the locationManager and locationListener, checks permissions and also gets location ;
I recommend you to use different methods for these operations ;
- **Comments**
I really like them but for god's sake start a phrase with a capital letter!
I've understood almost everything reading them, so good job ;

Overall, in a scale from 1 to 10, I give you a 6 but you can easily jump to 8 without these simple mistakes.

File: app/src/main/java/com/example/killerapp/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        Location location=locationManager.getLastKnownLocation(LocationManager.GPS_PROVIDER);
+        assert location != null;
+        gpsCoordinates.append("\n "+location.getLongitude()+"\n "+location.getLatitude());
+    }
+
+
+    /***
+     * when a message is received with only the key word it open a new empty activity,start an alarm and send a message ,to the phone number
+     * he received the message from ,with the device's current gps coordinates
+     * @param message Received SMSMessage class of SmsHandler library
```
Comment (Fabio-Marangoni) [inline app/src/main/java/com/example/killerapp/MainActivity.java]: Il codice è scritto molto bene e non ho nulla da aggiungere se non una piccola nota per questa specifica sull'uso dell'inglese (per esempio "opens" alla prima riga per la terza persona singolare, oppure "it" al posto di "he" all'inizio della seconda riga, essendo il telefono un oggetto).

**Comment by kummy98** [comment]
> * **Miscellaneous**
>   Some access modifiers are superfluous like "private" ;
>   You should also keep attention to constants using the caps lock (line: 36, 37, 38) ;
>   The keyword "<#>" is hardwired (line: 163) ;
> * **Methods**
>   getCoordinates (line: 118) does too much things (3): it instantiates the locationManager and locationListener, checks permissions and also gets location ;
>   I recommend you to use different methods for these operations ;
> * **Comments**
>   I really like them but for god's sake start a phrase with a capital letter!
>   I've understood almost everything reading them, so good job ;
> 
> Overall, in a scale from 1 to 10, I give you a 6 but you can easily jump to 8 without these simple mistakes.

Thanks for the suggestions, i can say that i agree with everything u've written.
I'll change the file as u suggested.

File: app/src/main/java/com/example/killerapp/MainActivity.java (hunk 1 above)
Comment (kummy98) [reply app/src/main/java/com/example/killerapp/MainActivity.java]: Thanks for the suggestion i'll try to check a bit better my comments next time

**Pull request merged**